tests/ChatTests.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE PostfixOperators #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
I don't think it's used here?

Reply:
it's used here, see DefaultChatPrefs in that file